src/PluralRules/Nl.elm

```diff
@@ -0,0 +1,90 @@
+module PluralRules.Nl exposing (pluralize, pluralizeFloat)
+
+{-| Dutch is simple wrt. pluralization: it has just 1 vs "not 1".
+The main gimmick of this module is adding `"en"` to the word unless your provided
+`Rules` dictionary says it should behave differently.
+
+
+# Usage
+
+Create your own dictionary of "non-s" pluralizations and a helper function that
+provides that dictionary to the `pluralize` function. Here's an example (see also
+the `examples/` folder):
+
+    module Pluralize exposing (pluralize)
+
+    import PluralRules exposing (Cardinal(..), Rules)
+    import PluralRules.En
```

Comment:
I wonder if we want to have such example in each of the language modules. Currently the English one has it but the Czech one doesn't.

Do you think it would be good to have it everywhere? Or should we just point people to `examples/`? Or something else?

In any case, if the examples are here to stay, perhaps let's change the code example to use `import PluralRules.Nl`, `PluralRules.Nl.pluralize` and some irregular (non-`-en`) Dutch word in the `rules` dictionary :slightly_smiling_face:

Reply:
I think the doc should be as barebone as possible for most languages. You only need english (very simple) , and Czesh (much complication :D) and you can only refer to them as better examples. We can add more doc later, should it be needed.